packages/cli/src/commands/dataMigrate/up.js

```diff
@@ -155,9 +155,11 @@ export const handler = async () => {
 
   try {
     await tasks.run()
+    process.exit(0)
```

Comment:
I think this would prevent the finally block, I don't think exiting here is the right thing to do.

Reply:
Hmmm you're right!